gnupg.py

```diff
@@ -723,6 +723,35 @@ def handle_status(self, key, value):
         else:  # pragma: no cover
             logger.debug('message ignored: %s, %s', key, value)
 
+class AddSubKey(object):
+    "Handle status messages for --quick-add-key"
+
+    returncode = None
+
+    def __init__(self, gpg):
+        self.gpg = gpg
+        self.type = None
+        self.fingerprint = None
+        self.status = None
+
+    def __nonzero__(self):
+        if self.fingerprint: return True
+        return False
+
+    __bool__ = __nonzero__
+
+    def __str__(self):
+        return self.fingerprint or ''
```

Comment:
You could set `self.fingerprint = ''` in `__init__()` and just do `return self.fingerprint` here.

Reply:
Same here, just copied it from `GenKey` and thought it would be fine :)

Reply:
It's not a real problem, but maybe I'll change those places too ...

Reply:
Sometimes I look at my old code and think that I would do it differently now.